src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
```

Comment:
@AGaliuzov, hide headers: fstream, from_stream. Check the rest of the boost headers. I think we can leave only one.

Add logger.cc and move impl there
create some
```
int load_log_config() {
  ....
}
```
and in header just call it.